sessions/delay-distributions.qmd

```diff
@@ -236,8 +236,23 @@ ggplot(res_df, aes(x = x, y = density, group = .draw)) +
 
 # Going further
 
--   In this session we were in the enviable situation of knowing which distribution was used to generate the data. With real data, of course, we don't have this information available. Try using a different distribution for inference (e.g. normal, or gamma). Do you get a good fit?
+## Challenge
+
+-   In this session we were in the enviable situation of knowing which distribution was used to generate the data. 
+With real data, of course, we don't have this information available. 
+Try using a different distribution for inference (e.g. normal, or gamma). 
+Do you get a good fit?
+
+## Methods in practice
+
+-   Charniga et al., [Best practices for estimating and reporting epidemiological delay distributions of infectious diseases using public health surveillance and healthcare data](https://doi.org/10.48550/arXiv.2405.08841) summarises challenges in estimating delay distributions into a set of best practices, with a practical flowchart and reporting checklist.
```

Comment:
do we want this here or in the biases one?

Reply:
I was adding here to flag it early on, but don't feel very strongly

Reply:
I think we should push to bias if that is okay as otherwise the content will all be new to people